Sources/SafeDICore/Visitors/InstantiableVisitor.swift

```diff
@@ -138,16 +138,10 @@ public final class InstantiableVisitor: SyntaxVisitor {
     }
 
     public override func visit(_ node: ExtensionDeclSyntax) -> SyntaxVisitorContinueKind {
-        guard declarationType.isExtension else {
-            return .skipChildren
-        }
```

Comment:
really not necessary. Extensions can't be nested in other types, and we only call the `InstantiableVisitor` with an `ExtensionDeclSyntax` on extensions.